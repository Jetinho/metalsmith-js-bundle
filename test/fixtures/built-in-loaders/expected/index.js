(()=>{var e="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAACdCAYAAABLlH3/AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAAVdEVYdFRpdGxlAE1ldGFsc21pdGggbG9nb4EGye0AAAAXdEVYdEF1dGhvcgBJYW4gU3Rvcm0gVGF5bG9y2odu4gAAAFJ0RVh0Q29weXJpZ2h0AENDIEF0dHJpYnV0aW9uLVNoYXJlQWxpa2UgaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbGljZW5zZXMvYnktc2EvNC4wL8NUYgUAAArASURBVHic7Z3ddhu3Ecf/WJKirPpITtK0keQ4tMSQlPfYNzqn17ruE/QV+xZ6Ad3YZvhhUnYSRVHStIlbhYrF3UUvSDYrmuLHDDAD1/7f5JzYWID+4WOAGQxMrV63kJS1f+t2u38XrXMF1er1MwC7UvVFUhX9XmN0IF7nkqpWq5sAdiTrlAdgbbAAUCo9AmAkqxQHYIFH0nUuK5Nl4p1DHIABakdHR0XpepdRBPz/AwCwdnFxsadQ70JZhelRA4DKD11KxohPjx8AjFWpVNYBfCFdrwqAEE3RUqnUAFCQrlcHgLXhWUKFgkqn0AEANCBsby+U0rSoBeDuXhx/rlT3bXqvAGAtTYNaB4yCBQQoAgjJEjo6OirC2n2Nuj8AAHB+fl4FUNaoWw2AxqZnjtQ6gx4AxR89LavYGTQBfLK/v/8nxfrzei9HAEypFMQo0DgFzdWtJxPGQhxZoK5WuVbF48rVATQajQcA/qBVvyoAzcVvokzBC5aXKoAg/MNRpNoJdAEAu3t7e1uqLVDuBNoAUCgUGspNeL8BaE8BGB2Nq0kdgKYlFMfxZwA+1qofcAfgF2pBzUO5JEk4dZN/c15OABigRS9s1ABYnmv0KxdtcALA8hrz8P79+3dctGNl8YIDwgEAazmNiTY2NmpO2rGq6NPfv2HtmYsmOAEQRVGTU97qWUK0enkd7oacAEiShAVAYzNUqVTuAfiMVNgY3u/NyQmAcrn8C4DvGZ8QB7C+vk4fdaGNAACsXqFxLM0xf40x4QEwnF5hTO3w8LDkqi3LiAOAPeXm5AxAxusVpcvLS9mQdfr+47Lf7zuxgACXIyDLWMOSuSmiiFbfaKp1drHRGYByufyc9QHBiOmdw8MNAA8oZVlT7Qw5A/Ds2bOfAVyQPyC4EG8MBg0Qfztzqn1Lrk9DOY0TA1BguCFNljlbgAHXAKylm6IjAFLH42QAhUIh3BFgGcPTAhvjCAXvYgQDXLZarW9ctsVtj2NaQmIRCvT15is4tIAAxwAKhQJ3fvQOYLzho4WiO7aAAMcA2u32P2HtD+QPCDhnXr9+/SUA0q6bM8XeJveLHq+R3jdjplikQ2ZOsbP03gGIGDtuB1PsW3IPgNdL7o0jFbyJeghngEG73XZqAQEeAGRM7xgzUmGxiOvM2O+dOW6NewARc6foOUwlMgDN/8zYZM6TcwDdbvcnAP9gfMIbgFqtVrHABqWsSydMXl62/hag9xaPIeuc0eX6EG4iLwBMqIdyjOiL6F2ZggBwd4zbjx8//shZW3KixqEaYNDpdL523R7A1whgDter4dBLxDJ1CrKj0EvnFhDgCQDXaV3wZQkZQwXrZf4HPAHo9/s/AviJ8QnnAOr1+g6Ae6TCHg7hJvLnAOH5BpxbQhynP3dzOU/eAFie1eB8BFiO0z9J3r0RwFyIv3jy5Inbu7v0deWq1+u9ctmUvPwBYIasDwYD1yHrVBO0BSB13Jb/yRuA61KJNW+yzu1nfY+4rjAvnyyUNwAvnz//AQxLKHJoisZx/DEAWmYWT0cQE/kOAyHfHXMZqjgcDsnfch0JNy2vAFhnQm79w/RI6GLRmwkKeAaQ8ebPahzHa04aQjdBf+s3my+dtOEW+R0BvL1A6fr62k0mQ/p60oZHCwjwDIAdxlcoOFkHGBaQ1+kH8Ayg1Wp9D+Bf5A84sITiOL5rAVKWXl9esLwkgmHJP8JFHokkSeh5qj3EAU3LPwCGGWcdWEIcN6SPOKBpeQfAGsbWki9S5ES2gFqt1imz7oXyDiCLIs4wvlOv13mvWtCd/B14toAAAQDcOKEsy7iWEG0EeN4BT+QdQKfTOQfwM/kDjJctxhs50vVXH5HQsyR1JYieT4gTy5NlNQCkN8uYm8ilJQOAkcaAY4pybtxI7AEAIQCcE0ULxOSy9BPVN9vb231qvatIBEDK602bBwcH26SS9H1E5/j4OCGWXUkiALiWUJqm1J5MAyA0/QBCALrd7nfgZBmkHScXAHxJqc63EyYvsbyhrMyKBEtoP44fAiAlA8wcZsRaJDEAzKPdlQEUk4TuhhQ4hJtILnOutZwRQPnHpC7A1zs7Oz1i2ZUlBoCVWdGYPzcajU9WKcKIhO5KWUCAIIDhcMhN6LRaZDPRBDUCXrC8xACM03y9ppZfsUcbULOiC1pAgGz2dMtaB1YwRavV6i6ATUo1UkcQE4mmr2edMK6yEBeL9FB0xwmZFkkUALN3LT0CGDdshpubm2IWECANgHfE+yCO47tL/l2qE6Z7cnIyJJUlShTAsFjkjACTpulSD66RnfmCO+CJRAGcNpvfAvgPtfzSlhD1GNoYupFAlPQbMha8iOmFAMYbtj9Svi/lBctL/BEf1kZniQiHNE3JDpzUcUbEZSQOgJlzYfEURHfiD8vGvCCWJUt+BPBOGver1er8p8epJqgxL5rN5jWpLEPiAJjhfkWsrVXn/QVDXICt8BHEROIAxolPyZZQIU3n93DqIZyCCQrovKRnMbr4QCs8p4ePN2q7lO9KuiHz0nnKkGPuzTmUG1/GI4WiR7wYVrJUABjOhmfeIku/C5YMh0NxCwhQAsA8caxjFPHwtuiHcL1er/eG3CKGVAAwh/t6tVqtzPoD6l0waS9YXioAOp3OKwCX1PK3pTGg+oGZ12lZ0npPmGUJzUo/XKlU1mHMQ8r3tCwgQPNBZ05Cpxk9vVwu13Db2rBA3LcwOVIDwOp1M6wdxmW8JEmSLrktTKkBYIX/jf6xb9r7dBO0r2UBAZojgHcot1mr1XZu/B+qE0bBB5CXGoBut/sKwK/U8nYqjYGlZsQSDkOZlt4iDGRmdBWUWPr360dHR0dFQwxF95UTellpAnAWMX1xcbEHYL6f4BZJRkLPkioAThhg/vIe4y5xOhgM1CwgQBkAZ/69kdyVYQGdnZ1dUdvgQqoA0jTlTEGf1mq1UfQD72E2VakC6PV6Lw0wYHziYOq/K0nzEG4i3TUAyKwxdEtoNPWQQ9G1LSBAHwBrI2StPdiL488BLBszelNp+gEAMxzwoES/jJddXV3RR58jqQPghAMa4BH5EM7aU20LCAgAANM7dh9R9BdSSaUwlGmpAxinBaP2RANr/0oqqeiEyUsdAEZpwThz8TtxF+w2hQCA++4YScxNoDMFAUDBKZ4NtrbULSAgEAAKFyNenp+ccHbgzhQEAOmwQImc0MsqCADtdrsP4Dep+kJZgIFAAGBkCcmdyys7YfIKBYCoc1wrEnqWggEglSgVQLa2tkaOynOtYAAIJsp79fTpU3I0hmsFA8DXg8lvKZAjiImCATBOlOo/Qi2QQ7iJggEwThPm3xL6MALmSKB3hrQHAEIDkGW+k2XYYrEYjAUEBAbA+o/T/7rZbJJv5vhQUAC8O8kDW4CBwADs7u6+gEdLSPMq0m0KCsDx8XFiAW/3dUOIA5pWUAAAv94xI5wRcRkFB8CjnW6NMUFZQECAADxmLfmm0+mQs7T4UnAAUn+WUHDTDxAggK2trRcAnGeu0oi8WEbBATg5ORnCGOfZazXTEcxTcAAA+PGOBeSGzCtMAO7tdZumaXAWEBAoANc7VgN8e3p6Sn67wKeCBJA6fkjZBjr/A4EC2LpzpwvAXRZz5XQE8xQkgHEKeWeWUGhOmLyCBAC4DR9MA4oDmlawAFz22uzNG/G09Mvqv89Ar0p2TtCMAAAAAElFTkSuQmCC";var o='<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xml:space="preserve" enable-background="new 0 0 745.199 161.769" viewBox="0 0 192.00001 192" height="192" width="192" y="0px" x="0px" version="1.1"><title>Metalsmith logo</title><metadata><rdf:RDF><cc:Work rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title>Metalsmith logo</dc:title><cc:license rdf:resource="http://creativecommons.org/licenses/by-sa/4.0/" /><dc:contributor><cc:Agent><dc:title>Kevin Van Lierde</dc:title></cc:Agent></dc:contributor><dc:creator><cc:Agent><dc:title>Ian Storm Taylor</dc:title></cc:Agent></dc:creator></cc:Work><cc:License rdf:about="http://creativecommons.org/licenses/by-sa/4.0/"><cc:permits rdf:resource="http://creativecommons.org/ns#Reproduction" /><cc:permits rdf:resource="http://creativecommons.org/ns#Distribution" /><cc:requires rdf:resource="http://creativecommons.org/ns#Notice" /><cc:requires rdf:resource="http://creativecommons.org/ns#Attribution" /><cc:permits rdf:resource="http://creativecommons.org/ns#DerivativeWorks" /><cc:requires rdf:resource="http://creativecommons.org/ns#ShareAlike" /></cc:License></rdf:RDF></metadata><g transform="translate(0,30.231003)"><path fill="#262727" style="fill:#262727;fill-opacity:1" d="M 96.267236,113.7268 117.50235,-17.740613 h 29.58592 V 149.27758 H 128.47749 V 11.13015 L 105.57158,149.27863 H 86.007257 L 63.102404,11.13015 V 149.27863 H 44.968389 V -17.740613 h 30.062685 z" /></g></svg>';var s="value";var n={};var m="./lib/square.jpg";document.getElementById("app").style.cssText="";})();
