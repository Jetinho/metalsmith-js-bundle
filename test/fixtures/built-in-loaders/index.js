import logo from './lib/logo-compact.png'
import svg from './lib/logo-compact.svg'
import { key } from './lib/data.json'
import style from './lib/style.css'
import square from './lib/square.jpg'

export {
  logo,
  svg,
  key,
  style,
  square
}

document.getElementById('app').style.cssText = ''