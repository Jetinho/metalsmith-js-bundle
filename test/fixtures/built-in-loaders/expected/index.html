<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
  </head>
  <body>
    <div id="app" style="display: none;">It's working!</div>
    <script src="index.js"></script>
  </body>
</html>